### DISCUSSION TITLE
EKS 사용 시 pipelines localhost 접속 질문

### DISCUSSION THREAD

**Discussion by ChaejinE** [mlops-for-all/mlops-for-all.github.io#114] [Q&A] [answered]
안녕하세요.

제가 ```kubectl port-forward svc/ml-pipeline-ui -n kubeflow 8888:80```진행 중에 pod가 모두 정상적으로 올라갔습니다.

그런데 사용중 제가 localhost:8888/#/pipelines 로 들어갔더니 정상 작동을 하던 것이 이상합니다.

![image](https://user-images.githubusercontent.com/69780812/212248592-2e409d25-c7a0-46d9-bc95-0301008dc7be.png)

분명히 EC2와 같은 공인 ip를 넣어야되는 걸로 이해했는데 말이죠.. localhost에서 들어가집니다..

해당 실행되는 노드는 EKS에서 관리하는 노드 중 하나일 것일 텐데.. 제 localhost url에서 작동하는게 이해가 안됩니다.. 이게 맞나요 ?

kubeflow yaml을 확인해봐도 이해가 안가서 질문드립니다.

**Comment by anencore94** [comment, marked as answer]
`kubectl port-forward` 의 동작 방식에 대해서 찾아보시면 이해하실 수 있을겁니다 :)

**Reply by ChaejinE** [reply]
![image](https://user-images.githubusercontent.com/69780812/212253776-b838da3f-f2ed-4dd3-b24a-7623df54f531.png)

감사합니다.
